src/Styleguide/Components/Tabs.tsx

```diff
@@ -74,8 +92,17 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
 }
 
 interface TabProps {
+  /** Display name of the Tab */
   name: string
+
+  /**
+   * Arbitrary data that can be associated with a Tab.
+   *
+   * Will be passed to the parent <Tabs>'s onChange handler.
+   */
+  data?: object
```

Comment:
If this should truly be any type of data, then maybe this needs to be `any` so scalars like numbers are supported too.

Reply:
Good call, fixed in fdc609ce252ea185a3771e5160bf5257eda87f76